plugins/janus_streaming.c

```diff
@@ -1132,6 +1137,10 @@ typedef struct janus_streaming_mountpoint {
 	gboolean audio, video, data;
 	GList *viewers;
 	int helper_threads;		/* Only relevant for RTP/RTSP mountpoints */
+	gint64 reconnect_delay; /* Only relevant for RTSP mountpoints */
+	gint64 session_timeout; /* Only relevant for RTSP mountpoints */
+	int rtsp_timeout; /* Only relevant for RTSP mountpoints */
+	int rtsp_conn_timeout; /* Only relevant for RTSP mountpoints */
```

Comment:
These properties do not belong here. A `janus_streaming_mountpoint` instance is a generic container for very different mountpoint categories, and RTSP is only one of those. The new properties should be where the others are, so in `janus_streaming_rtp_source` and in the `#ifdef HAVE_LIBCURL` block.

Reply:
I moved it to janus_streaming_rtp_source structure.